Code/Tools/ProjectManager/Resources/ProjectManager.qss

```diff
@@ -453,12 +453,60 @@ QTabBar::tab:focus {
     margin-top:6px;
 }
 
-#projectButton > QFrame > QLabel {
+#projectButton QLabel {
```

Comment:
I think you need to be careful here because I seem to recall this was done to prevent every label from becoming bold - just the labels inside the QFrame.  Have you verified visually this looks correct?